actionview/lib/action_view.rb

```diff
@@ -48,6 +48,7 @@ module ActionView
     autoload :TemplatePath
     autoload :UnboundTemplate
     autoload :ViewPaths
+    autoload :ERBTracker
```

Comment:
Should it be inside `eager_autoload` given it is not used?

Reply:
🤦 It should not. Thanks!